src/pudl_archiver/depositors/zenodo.py

```diff
@@ -122,26 +127,52 @@ async def create_deposition(self, metadata: DepositionMetadata) -> Deposition:
         Returns:
             The brand new deposition - unpublished and with no files in it.
         """
-        url = f"{self.api_root}/deposit/depositions"
+        url = f"{self.api_root}/records"
         headers = {
+            "Accept": "application/vnd.inveniordm.v1+json",
             "Content-Type": "application/json",
         } | self.auth_write
 
-        payload = {
-            "metadata": metadata.dict(
-                by_alias=True,
-                exclude={"publication_date", "doi", "prereserve_doi"},
-            )
-        }
-
+        # Set up metadata from data source
+        metadata = metadata.dict(
+            by_alias=True,
+            exclude={"publication_date", "doi", "prereserve_doi"},
+        )
+        metadata["publication_date"] = str(date.today())
+
+        payload = json.dumps(
+            {
+                "access": {
+                    "record": "public",
+                    "files": "public",
+                },
+                "files": {"enabled": True},  # Must always be True for Zenodo
+                "metadata": metadata,
+            }
+        )
         response = await self.request(
-            "POST", url, "Create new deposition", json=payload, headers=headers
+            "POST", url, "Create new deposition", data=payload, headers=headers
         )
+        # Reserve DOI for deposition
+        doi_response = await self.reserve_doi(response)
```

Comment:
I followed the `"reserve_doi"` link returned in the list of links for a file. It's not listed anywhere other than being described as an option for developers using InvenioRDM to enable. Made some informed guesses about what it would take based on [this publish method](https://inveniordm.docs.cern.ch/reference/rest_api_drafts_records/#publish-a-draft-record) that wound up being right, if I remember right.